lib/xcode.js

```diff
@@ -109,7 +109,8 @@ async function getVersionWithoutRetry () {
 
   // we want to read the CFBundleShortVersionString from Xcode's plist.
   // It should be in /[root]/XCode.app/Contents/
-  const plistPath = xcodePath.replace(XCODE_SUBDIR, "/Contents/Info.plist");
+  let plistPath = xcodePath.replace(XCODE_SUBDIR, "/Contents/Info.plist");
+  plistPath = plistPath.replace(new RegExp("/$"), "").trim();
```

Comment:
Definitely we need to rewrite that.

But just in this line you are reinventing the wheel, you can just do:
```
path.join(plistPath, '.')
```